Sources/CryptorRSA/CryptorRSA.swift

```diff
@@ -438,23 +447,34 @@ public class CryptorRSA {
             defer {
                 EVP_CIPHER_CTX_reset_wrapper(rsaEncryptCtx)
                 EVP_CIPHER_CTX_free_wrapper(rsaEncryptCtx)
-                EVP_PKEY_free(evp_key)
             }
             
             EVP_CIPHER_CTX_set_padding(rsaEncryptCtx, padding)
             
             // Initialize the AES encryption key array (of size 1)
             typealias UInt8Ptr = UnsafeMutablePointer<UInt8>?
             var ek: UInt8Ptr
-            ek = UnsafeMutablePointer<UInt8>.allocate(capacity: Int(EVP_PKEY_size(evp_key)))
+            ek = UnsafeMutablePointer<UInt8>.allocate(capacity: Int(EVP_PKEY_size(.make(optional: key.reference))))
```

Comment:
Can you explain why this change is correct / necessary?

Reply:
Nevermind, I get it - key is (now) a pointer to an envelope key, so this is equivalent to what we had before